src/plugin_config.test.js

```diff
@@ -0,0 +1,88 @@
+const test = require('tape');
+const pluginConfig = require('./plugin_config');
+
+test('should return a default config', (assert) => {
+  assert.plan(6);
+
+  const config = pluginConfig.load();
+
+  assert.equal(config.consulAddr, 'https://127.0.0.1:8500');
```

Comment:
Could have used `assert.deepEqual` here, assuming you are checking all of the keys.

Reply:
There are functions on the config object, so `deepEqual` doesn't help here